components/tools/OmeroJava/test/integration/chown/PermissionsTest.java

```diff
@@ -19,11 +19,14 @@
 
 package integration.chown;
 
+import static omero.rtypes.rstring;
```

Comment:
I'm sure I've waffled on this. Happy to be told one or the other is "our way" and have it enforced externally :smile: